README.md

```diff
@@ -311,7 +311,7 @@ sudo easy_install pip
 sudo pip install -r python_requirements.txt
 ```
 
-If previous command fails because of gdal, comment the dependency: `# gdal==1.10.0`.
+If previous command fails because of gdal, comment the dependency: `# gdal==1.10.0`. Try `sudo easy_install GDAL` instead (GDAL is needed for SHP normalization, for example).
```

Comment:
This is obviously a better solution than commenting out the dependency :D

I have a few comments though:

1. It's preferable to install a specific version, to avoid update issues.
2. The default version at this time is 2.1
3. Maybe we could simply update the `python_requirements.txt` to use version 2.1.